index1.php

```diff
@@ -5,6 +5,36 @@
 <link rel="stylesheet" href="style.css">
 </head>
 <body>
+
+<?php
+
+if($_GET['q']==1){
+    echo "<script>location.href='index1.php'</script>";
+}
+elseif($_GET['q']==2){
+    echo "<script>location.href='index2.php'</script>";
+
+}
+elseif($_GET['q']==3){
+    echo "<script>location.href='index3.php'</script>";
+
+}
+elseif($_GET['q']==4){
+    echo "<script>location.href='index4.php'</script>";
+
+}
+elseif($_GET['q']==5){
+    echo "<script>location.href='index5.php'</script>";
```

Comment:
Please include file here instead of adding script here.

Reply:
added a landing page